test.js

```diff
@@ -351,6 +352,15 @@ describe("Satellite()", () => {
       expect(res.ok).toBe(false);
       expect(res.status).toEqual(401);
 
+      // Protected should work with correct authorization token
```

Comment:
Why this change?

Reply:
Oh dear, I think that was due to my editor, let me clean that up